src/installer.ts

```diff
@@ -82,15 +224,14 @@ export async function getNode(
   }
 
   // check cache
+  core.info('Attempt to find existing version in cache...');
```

Comment:
I think we can remove this line because we have a log to point if it was set up from cache. 
- https://github.com/actions/setup-node/blob/main/src/installer.ts#L98